lib/config.template.php

```diff
@@ -17,6 +17,7 @@ class Config {
   ];
 
   public static $pgpVerificationAPI = 'http://127.0.0.1:9009';
+  public static $redisAPI = 'tcp://127.0.0.1:6379';
```

Comment:
I thought about doing this in my work for docker to heroku, but then putting it here, it can be overwritten. If that behaviour is desirable then the place that constructs a redis client should have a null param, which if null borrows the value from ENV

Reply:
So this API URLs should also be configurable via env vars?

Reply:
Why not? A file needs the entire document re-built. An ENV variable takes just the value it needs

Reply:
It is now possible to set PGP and Redis APIs with env vars.